### PR TITLE
add default TableName of NewAdapterByDB()

### PR DESCRIPTION
**Pull request by miaolz123**

### CODE REVIEW

**Comment by miaolz123** [comment]
from #50

**Comment by hsluoyz** [comment]
@nodece @GopherJ @00LT00 please review.

**Review by 00LT00 — APPROVED** [review]
LGTM

File: adapter.go [hunk 1]
```diff
@@ -184,8 +184,8 @@ func openDBConnection(driverName, dataSourceName string) (*gorm.DB, error) {
 		db, err = gorm.Open(postgres.Open(dataSourceName), &gorm.Config{})
 	} else if driverName == "mysql" {
 		db, err = gorm.Open(mysql.Open(dataSourceName), &gorm.Config{})
-	//} else if driverName == "sqlite3" {
-	//	db, err = gorm.Open(sqlite.Open(dataSourceName), &gorm.Config{})
+		//} else if driverName == "sqlite3" {
```
Comment (hsluoyz) [inline adapter.go:187]: Why change it.

File: adapter.go [hunk 2]
```diff
@@ -148,14 +151,19 @@ func NewAdapter(driverName string, dataSourceName string,params ...interface{})
 	return a, nil
 }
 
-// NewAdapterByDB obtained through an existing Gorm instance get  a adapter, specify the table prefix and the table name
+// NewAdapterByDBUseTableName obtained through an existing Gorm instance get  a adapter, specify the table prefix and the table name
 // Example: gormadapter.NewAdapterByDBUseTableName(&db, "cms", "casbin") Automatically generate table name like this "cms_casbin"
-func NewAdapterByDBUseTableName(db *gorm.DB, prefix string,tablename string) (*Adapter, error) {
+func NewAdapterByDBUseTableName(db *gorm.DB, prefix string, tablename string) (*Adapter, error) {
```
Comment (hsluoyz) [inline adapter.go:156]: Rename to `NewAdapterByDBWithTableName()`, arg to `tableName`

File: adapter.go [hunk 3]
```diff
@@ -148,14 +151,19 @@ func NewAdapter(driverName string, dataSourceName string,params ...interface{})
 	return a, nil
 }
 
-// NewAdapterByDB obtained through an existing Gorm instance get  a adapter, specify the table prefix and the table name
+// NewAdapterByDBUseTableName obtained through an existing Gorm instance get  a adapter, specify the table prefix and the table name
```
Comment (hsluoyz) [inline adapter.go:154]: `obtained through an existing Gorm instance get  a adapter`

This sentense doesn't make sense. Change it to smooth English.

File: adapter.go [hunk 4]
```diff
@@ -164,17 +172,9 @@ func NewAdapterByDBUseTableName(db *gorm.DB, prefix string,tablename string) (*A
 	return a, nil
 }
 
+// NewAdapterByDB obtained through an existing Gorm instance get  a adapter
```
Comment (hsluoyz) [inline adapter.go:175]: `obtained through an existing Gorm instance get  a adapter`

This sentense doesn't make sense. Change it to smooth English.

**Comment by hsluoyz** [comment]
@miaolz123 reviewed.

File: adapter.go (hunk 1 above)
Comment (miaolz123) [reply adapter.go:187]: The change of these lines's is just for code format.

File: adapter.go (hunk 2 above)
Comment (miaolz123) [reply adapter.go:156]: Also, the change of this line is just for code format and I have no plans to unify code-style at this PR.

**Comment by miaolz123** [comment]
To be honest, I submitted this PR only to fix default table name bug which make the access-control of my project doesn't work right.

Due to the limited free time, I have no plans to unify code-style and commet-spell at this PR.

please close this PR if you think my modified code is unacceptable. I can accept it totally.

**Comment by hsluoyz** [comment]
This PR is acceptable. I just found some other places that can be improved. I will find others to finish what I proposed. Thanks.

**Comment by hsluoyz** [comment]
@00LT00 please send a new PR to fix what I have commented in this PR.

**Comment by 00LT00** [comment]
ok

**Comment by hsluoyz** [comment]
@00LT00 you can first create an issue to track this.

**Pull request merged**